lib/src/io/node.dart

```diff
@@ -48,6 +58,16 @@ class Stderr {
   void flush() {}
 }
 
+class Stdin {
```

Comment:
What's the benefit of defining this class, instead of just using `_Stdin` in `readStdin()`?